lib/pantry/alexa/verification_error.rb

```diff
@@ -0,0 +1,17 @@
+require 'uri'
+
+module Pantry
+
+  module Alexa
+
+    class VerificationError < StandardError
+
+      def initialize(msg="Invalid Alexa Request")
+        super msg
+      end
```

Comment:
Ahhh, I see.  You're using that as a shorthand way to do:
```ruby
raise VerificationError, 'Invalid Alexa Request'
```

Hmm, I suppose this works, but it's a little opaque.